front/.env

```diff
@@ -1 +1,2 @@
 VUE_APP_BASE_URL=http://localhost:8000/api/
+VUE_APP_WEBSOCKET_URL=ws://localhost:8000/ws
```

Comment:
It's possible but then mroe difficult to maintain, If the url is hard coded then we have to change the code if we want to put `https/wss` rather than `http/ws` and if we change the API endpoint (add a v1 or v2..) we have to change the code at multiple locations again rather than just the configuration file.
So it's simpler like that and doesn't seem weird to have urls in a config file

Reply:
We might also add other urls later (maybe a link to a wiki) and I think it's better to just keep it simple with all the urls in one file rather than dispatched to other places

Reply:
Nah you take the host/port/protocole somewhere in the configuration file (letting the user pick https/http or wss/ws), and then build the URL somewhere else, and use that built version everywhere in the code. Still no hard coded information anywhere and a simpler configuration file.

Reply:
Done